Makefile

```diff
@@ -1,14 +1,14 @@
 start: restore backup
 
 restore: db
-	docker-compose up -d restore
+	docker compose up -d restore
```

Comment:
I suppose this is the better way forward. See green comment, before showing how to do it the old way.
https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-22-04

Reply:
It looks like it is the only way forward.

> From July 2023 Compose V1 stopped receiving updates.
> (https://docs.docker.com/compose/migrate/)

Reply:
Really? I was able to download a newer version of "docker-compose" matching "docker compose"s version. Though, the space version is easier, I think.

Reply:
Agreed, and if that's the direction Docker is going, I'll follow along until they make it too hard to do so.